crowd_anki/export/note_sorter.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+
+from ..config.config_settings import ConfigSettings
+
+DEBUG = False
+
+@dataclass
+class NoteSorter():
+    sort_method: list
+    is_reversed: bool
+    skip_sorting: bool
+    sort_key_tuple: tuple
+
+    @classmethod
+    def from_config(cls, config: ConfigSettings):
+        sorting_definitions = {
+            ConfigSettings.DeckExportSortMethods.NO_SORTING: None,
```

Comment:
this looks combersome, we're probably better off extracting the enum from the configsettings class to reduce the noise here.

Reply:
I meant extracting the DeckExportSortMethods to be a top-level enum and not be part of ConfigSettings

Reply:
Done.